packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -174,9 +174,6 @@ watch(
     color: darkred;
     margin: 0;
   }
-  &__info {
-    color: grey;
```

Comment:
was the contrast not good enough anymore? :(

Reply:
Nope :D